internal/cmd/local/command_kafka_start.go

```diff
@@ -127,46 +127,109 @@ func (c *Command) kafkaStart(cmd *cobra.Command, args []string) error {
+
+	// create a customized network
+	_, err = dockerClient.NetworkCreate(
+		context.Background(),
+		"confluent-local-network",
+		types.NetworkCreate{
+			CheckDuplicate: true,
+			Driver:         "bridge",
+		},
+	)
+	if err != nil && !strings.Contains(err.Error(), "already exists") {
+		return err
+	}
+
+	config1 := &container.Config{
 		Image:    dockerImageName,
-		Hostname: "broker",
-		Cmd:      strslice.StrSlice{"bash", "-c", "'/etc/confluent/docker/run'"},
+		Hostname: "broker1",
+		Cmd:      containerStartCmd,
 		ExposedPorts: nat.PortSet{
-			natKafkaRestPort: struct{}{},
-			natPlaintextPort: struct{}{},
+			natKafkaRestPorts[0]: struct{}{},
+			natPlaintextPorts[0]: struct{}{},
 		},
-		Env: getContainerEnvironmentWithPorts(ports),
+		Env: getContainerEnvironmentWithPorts(ports, 0),
 	}
-	hostConfig := &container.HostConfig{PortBindings: nat.PortMap{
-		natKafkaRestPort: []nat.PortBinding{
-			{
-				HostIP:   localhost,
-				HostPort: ports.KafkaRestPort,
+	hostConfig1 := &container.HostConfig{
+		NetworkMode: container.NetworkMode("confluent-local-network"),
+		PortBindings: nat.PortMap{
+			natKafkaRestPorts[0]: []nat.PortBinding{
+				{
+					HostIP:   localhost,
+					HostPort: ports.KafkaRestPorts[0],
+				},
+			},
+			natPlaintextPorts[0]: []nat.PortBinding{
+				{
+					HostIP:   localhost,
+					HostPort: ports.PlaintextPorts[0],
+				},
 			},
 		},
-		natPlaintextPort: []nat.PortBinding{
-			{
-				HostIP:   localhost,
-				HostPort: ports.PlaintextPort,
+	}
+
+	config2 := &container.Config{
+		Image:    dockerImageName,
+		Hostname: "broker2",
+		Cmd:      containerStartCmd,
+		ExposedPorts: nat.PortSet{
+			natKafkaRestPorts[1]: struct{}{},
+			natPlaintextPorts[1]: struct{}{},
+		},
+		Env: getContainerEnvironmentWithPorts(ports, 1),
+	}
+	hostConfig2 := &container.HostConfig{
+		NetworkMode: container.NetworkMode("confluent-local-network"),
+		PortBindings: nat.PortMap{
+			natKafkaRestPorts[1]: []nat.PortBinding{
+				{
+					HostIP:   localhost,
+					HostPort: ports.KafkaRestPorts[1],
+				},
+			},
+			natPlaintextPorts[1]: []nat.PortBinding{
+				{
+					HostIP:   localhost,
+					HostPort: ports.PlaintextPorts[1],
+				},
 			},
 		},
-	},
 	}
 
-	createResp, err := dockerClient.ContainerCreate(context.Background(), config, hostConfig, nil, platform, confluentLocalContainerName)
+	createResp1, err := dockerClient.ContainerCreate(context.Background(), config1, hostConfig1, nil, platform, "broker1")
+	if err != nil {
+		return err
+	}
+	log.CliLogger.Tracef("Create confluent-local container for broker 1 success")
+	err = dockerClient.ContainerStart(context.Background(), createResp1.ID, types.ContainerStartOptions{})
 	if err != nil {
 		return err
 	}
-	log.CliLogger.Tracef("Create confluent-local container success")
 
-	err = dockerClient.ContainerStart(context.Background(), createResp.ID, types.ContainerStartOptions{})
+	multiBroker, err := cmd.Flags().GetBool("multi-broker")
```

Comment:
I think that we should go for configurable N brokers (1 <= N <= 5 or 10 or something) rather than a flag that gives you 2. I suspect users will want to play with other numbers, with 2 and 3 (common replication factor 3), maybe 4 being the common cases